rust/flatbuffers/src/builder.rs

```diff
@@ -312,32 +308,6 @@ impl<'fbb> FlatBufferBuilder<'fbb> {
         WIPOffset::new(self.used_space() as UOffsetT)
     }
 
-    /// Create a vector of strings.
-    ///
-    /// Speed-sensitive users may wish to reduce memory usage by creating the
-    /// vector manually: use `start_vector`, `push`, and `end_vector`.
-    #[inline]
-    pub fn create_vector_of_strings<'a, 'b>(
```

Comment:
This method was only marginally faster than the naive approach, and whilst I did update it to use a combination of MaybeUnit and remaining_capacity_mut, the added complication did not seem to be worthwhile